main.py

```diff
@@ -1,10 +1,82 @@
+import json
+from typing import Union
+from django.db.models import Model
+
 import init_django_orm  # noqa: F401
 
-from db.models import Race, Skill, Player, Guild
+from db.models import Guild, Player, Skill, Race
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file_pointer:
+        data = json.load(file_pointer)
+
+    for player_name, data in data.items():
+        guild_data = data.get("guild")
+        race_data = data.get("race")
+
+        guild_obj = get_or_create_guild(guild_data)
+        race_obj = get_or_create_race(race_data)
+
+        for skill in race_data["skills"]:
+            get_or_create_skill(skill, race_obj)
+
+        player = Player.objects.create(
+            nickname=player_name,
+            email=data["email"],
+            bio=data["bio"],
+            race=race_obj,
+            guild=guild_obj
+        )
+        player.save()
+
+
+def get_or_create_skill(skill: dict, race_obj: Model) -> Union[Model, None]:
+    if not skill:
+        return None
+
+    skill_name = skill["name"]
+    if not Skill.objects.filter(name=skill_name).exists():
+        skill_obj = Skill.objects.create(
+            name=skill_name,
+            bonus=skill["bonus"],
+            race=race_obj
+        )
+
+    else:
+        skill_obj = Skill.objects.get(name=skill_name)
+
+    return skill_obj
```

Comment:
read about `get_or_create` method

Reply:
I suspected there had to be something here...😅